packages/easy-email-core/vite.config.ts

```diff
@@ -27,6 +27,10 @@ export default defineConfig({
         'react-dom',
         'react-dom/server',
         'lodash',
+        // Let uuid library handle how to use its crypto module depending on the environment
+        // Otherwise, uuid will be bundled into a specific environment (here it would expect to be in a browser environment)
+        // and any project importing easy-email-core in a node environment would fail with a missing 'crypto' module error
+        'uuid'
```

Comment:
If you do not specify the version of uuid in package json, it may cause the later version to be inconsistent

Reply:
Hello! I'm not sure I understand. The version is already specified in the package.json. I did not remove the dependency. 

This line is only to make sure the uuid package isn't bundled in the `index.cjs` or `index.es` module. It instead requires it in the node_modules.